README.md

```diff
@@ -0,0 +1,25 @@
+# apostrophe-link-widgets
+
+A link widget for [ApostropheCMS](http://apostrophecms.org/).
+
+`apostrophe-link-widgets` provides schema and a view for links.
```

Comment:
This sentence is awkward and very inside baseball. How about:

`apostrophe-link-widgets` provides a simple way for users to create a link, giving them the option to choose a page or enter a URL manually.